azerothcore/azerothcore-wotlk
# fix(DB/GO): Relocate Peacebloom in Elwynn Forest to avoid collision with Darkmoon Faire

This is my first PR and was created with the help of @necropola.

## Changes Proposed:
-  Change the position of Peacebloom (guid=26813) slighty, so that there is no collision with gameobjects of the Darkmoon Faire event
-  Remove this Peacebloom (guid=26813) spawn from `game_event_gameobject` for the Darkmoon Faire event (this led to a one-minute respawn of the Peacebloom)

## Issues Addressed:
- Closes #6181
- Closes https://github.com/chromiecraft/chromiecraft/issues/757

## SOURCE:
- Nodes shouldn't clip with other GO and shouldn't respawn every minute.

## Tests Performed:
- Ran SQL, no errors
- Tested in-game, the Peacebloom does not respawn every minute now

## How to Test the Changes:
1. .go go 26813
2. the Peacebloom does not always spawn
3. check with .gobject near 5, that there is indeed a spawn location for the mentioned Peacebloom and no other GO close to it.
4. if the peacebloom is spawned (reset server), take it and see, that it does not respawn after one minute (pooling does work correctly now for this Peacebloom)

## Known Issues and TODO List:


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
